README.md

```diff
@@ -436,7 +436,13 @@ import (
 
 func main() {
 
-	config := openai.DefaultAzureConfig("your Azure OpenAI Key", "https://your Azure OpenAI Endpoint ", "your Model deployment name")
+	config := openai.DefaultAzureConfig("your Azure OpenAI Key", "https://your Azure OpenAI Endpoint")
+	config.AzureModelMapperFunc = func(model string) string {
+        azureModelMapping = []string{
```

Comment:
Thank you for the PR! Could you please comment out the `config.AzureModelMapperFunc` section and clearly indicate that it is optional?

Reply:
Of course, great idea